applications/convection_diffusion_application/python_scripts/convergence_criteria_factory.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Check that applications were imported in the main script
+KratosMultiphysics.CheckRegisteredApplications("ConvectionDiffusionApplication")
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Convergence criteria class
+class convergence_criterion:
```

Comment:
Why is this in the convection-diffusion application? I see no application-specific things here...